src/test/java/uk/gov/ons/census/casesvc/messaging/RefusalReceiverIT.java

```diff
@@ -114,7 +111,7 @@ public void testRefusalEmitsMessageAndLogsEventForCase()
 
     Case actualCase = caseRepository.findById(TEST_CASE_ID).get();
     assertThat(actualCase.getSurvey()).isEqualTo("CENSUS");
-    assertThat(actualCase.isRefusalReceived()).isTrue();
+    assertThat(actualCase.getRefusalReceived()).isEqualTo(RefusalType.HARD_REFUSAL.toString());
```

Comment:
so much better to write `assertThat(actualCase.getRefusalReceived()).isEqualTo(RefusalType.HARD_REFUSAL);`

not using enums is against coding standards for these reasons